src/app/common/markdown-editor/markdown-editor.scss

```diff
@@ -22,7 +22,7 @@ $markdown-editor-height: 15px;
     }
   }
 
-.markdown-editor .markdown-editor-inner-context {
+.markdown-editor, .markdown-editor-inner-context-text, .markdown-editor-inner-context-audio {
```

Comment:
Shouldn't context-text and context-audio be nested inside markdown-editor as it was earlier?

Reply:
I have nested context-audio, context text has no unique styling hence its placement.